go.mod

```diff
@@ -1,17 +1,17 @@
 module github.com/prometheus/common
 
 require (
-	github.com/go-kit/log v0.1.0
+	github.com/go-kit/log v0.2.0
 	github.com/golang/protobuf v1.4.3
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/matttproud/golang_protobuf_extensions v1.0.1
 	github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f
 	github.com/prometheus/client_golang v1.11.0
 	github.com/prometheus/client_model v0.2.0
-	golang.org/x/net v0.0.0-20210525063256-abc453219eb5
-	golang.org/x/oauth2 v0.0.0-20210514164344-f6687ab2804c
+	golang.org/x/net v0.0.0-20220105145211-5b0dc2dfae98
+	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 	gopkg.in/yaml.v2 v2.4.0
 )
 
-go 1.13
```

Comment:
https://github.com/prometheus/client_golang/blob/main/go.mod#L15 Please update client_golang first

Reply:
I made a PR against client_golang for that: https://github.com/prometheus/client_golang/pull/980

Reply:
This one can be resolved, as the PR got merged.